.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Description
+
+Please include a summary of the change.
+
+Fixes # (issue number, if there is one)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Checklist:
+
+- [ ] This branch is based on `develop`
+- [ ] This PR is targeted to merge into `develop` (not `master`)
+- [ ] I have formatted my code with `flutter format`
```

Comment:
I would add `and the CI is passing`

Reply:
Good idea, any suggestion of how to write that so that people not familiar with CI in github understands?

Reply:
Updated, what do you think now?